wp_localization/localization/tr-TR/main.ftl

```diff
@@ -22,6 +22,8 @@ http_authentication_required_error = {$url} adresindeki sunucu kimlik doğrulama
 
 http_authentication_rejected_error = {$url} adresindeki sunucu giriş bilgilerinizi reddetti. Lütfen geçerli bir kullanıcı adı ve şifre girin.
 
+http_server_error = Unable to connect to server: {$reason}. Please contact your server provider.
+
```

Comment:
```suggestion
```

Reply:
No it won't. It'll output a warning and lower the translation percentage. So, it's best to not include any untranslated strings here.